libraries/AP_ADSB/AP_ADSB.h

```diff
@@ -22,11 +22,18 @@
 */
 
 #include <AP_HAL/AP_HAL.h>
+#include <AP_HAL/AP_HAL_Boards.h>
 #include <AP_Common/AP_Common.h>
```

Comment:
these other includes should come in after the ```#if HAL_ADSB_ENABLED```

Reply:
You need boards / hal in order to get the right defines for board flash size and minimize features